File: 1050.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var DDD = Number(lines.shift());
if (DDD === 61){
    console.log('Brasilia');}
else if (DDD === 71){
    console.log('Salvador');}
else if (DDD === 11){
    console.log('Sao Paulo');}
else if (DDD === 21){
    console.log('Rio de Janeiro');}
else if (DDD === 32){
    console.log('Juiz de Fora');}
else if (DDD === 19){
    console.log('Campinas');}
else if (DDD === 27){
    console.log('Vitoria');}
else if (DDD === 31){
    console.log('Belo Horizonte');}
else{
    console.log('DDD nao cadastrado');}

File: 1050.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main(){
    int DDD;
    cin >> DDD;
    if(DDD == 61){
        cout << "Brasilia" << endl;
    }
    else if(DDD == 71){
        cout << "Salvador" << endl;
    }
    else if(DDD == 11){
        cout << "Sao Paulo" << endl;
    }
    else if(DDD == 21){
        cout << "Rio de Janeiro" << endl;
    }
    else if(DDD == 32){
        cout << "Juiz de Fora" << endl;
    }
    else if(DDD == 19){
        cout << "Campinas" << endl;
    }
    else if(DDD == 27){
        cout << "Vitoria" << endl;
    }
    else if(DDD == 31){
        cout << "Belo Horizonte" << endl;
    }
    else{
        cout << "DDD nao cadastrado" << endl;
    }
    return 0;
}
